### PR TITLE
removed Alphabet from Bio.SubsMat

### PR DESCRIPTION
**Pull request by mdehoon**
This pull request removes Bio.Alphabet from Bio.SubsMat, and replaces the alphabet attribute by a string of letters (same as in Bio.motifs for example).

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Bio/SubsMat/FreqTable.py [hunk 1]
```diff
@@ -86,8 +83,7 @@ def __init__(self, in_dict, dict_type, alphabet=None):
         else:
             raise ValueError("bad dict_type")
         if not alphabet:
-            self.alphabet = Alphabet.Alphabet()
-            self.alphabet.letters = self._alphabet_from_input()
+            self.alphabet = self._alphabet_from_input()
```
[the remark this answers is not in the document]
Comment (mdehoon) [reply Bio/SubsMat/FreqTable.py:86]: In the next release of Biopython, `Bio.Alphabet` won't exist any more.

[the remark this answers is not in the document]
Comment (peterjc) [reply Bio/SubsMat/FreqTable.py:86]: I think a deprecated stub to make the transition less painful is more sensible. Any hard breaks in compatibility are going to be unpopular and make more work in documentation and issue triage.

[the remark this answers is not in the document]
Comment (mdehoon) [reply Bio/SubsMat/FreqTable.py:86]: Deprecated stubs just prolong the agony and cause more confusion (and we have already seen examples of that recently). And again, we have decided to *remove* the alphabets.

[the remark this answers is not in the document]
Comment (mdehoon) [reply Bio/SubsMat/FreqTable.py:86]: The only thing I could offer you is to deprecate `Bio.SubsMat`. It is not being actively maintained, while the alternative in `Bio.Align.substitution_matrices` is and is more up to date.

**Pull request merged**